### ISSUE TITLE
Add more field types button creates confusion

### ISSUE THREAD

**Issue by ineagu** [Codeinwp/woocommerce-product-addon#145]
### Description

I was trying to use the plugin and explore various types of fields and click on the "Add more field types" button ( https://vertis.d.pr/i/ur7o4e) , which looked like it was part of product functionality, while instead, this was an upsell button, disguised as a product feature.

This creates frustration and confusion about how the product is supposed to work.

I think we can just remove that button, we're doing a great job to highlight available fields in context, when the user is trying to add a field, the button is not required.

### Step-by-step reproduction instructions

1. Go to the PPOM Fields page.

### Screenshots, screen recording, code snippet or Help Scout ticket

![Annotation on 2023-05-09 at 11-40-02](https://user-images.githubusercontent.com/1849868/237057851-f08ff03d-3f63-4450-9026-94e753300e65.png)

### Environment info

https://pastebin.com/WGVR0YEN

### Is the issue you are reporting a regression

No

**Comment by ineagu** [comment]
I was thoroughly checking the setup & upsell experience for this, looking only at the traffic coming from that can be misleading, since is normal to bring a lot of traffic if we disguise it like this, just that we don't want to do this at the expense of the product experience.

**Comment by selul** [comment]
@ineagu I agree, that's why I asked if it worth to be looking at the conversions from that.

**Comment by selul** [comment]
Here we can move forward with this on PRF.
